tenant-partitioning-config-service-api/src/main/proto/org/hypertrace/tenantpartitioning/config/service/v1/tenant_partition_group_config.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package org.hypertrace.tenantpartitioning.config.service.v1;
+
+option java_multiple_files = true;
+
+message TenantPartitionGroupConfig{
+  string id = 1;
+  string group_name = 2;
```

Comment:
what do we do with name? just for logging?

Reply:
most informational purpose